PWGJE/FlavourJetTasks/AliPicoV0.h

```diff
@@ -31,21 +31,29 @@ class AliPicoV0 : public TObject {
   virtual ~AliPicoV0();
 //=============================================================================
 
-  TVector3 KinePos() const { return  fP3Pos; }
-  TVector3 KineNeg() const { return  fP3Neg; }
-  TVector3 KineRD()  const { return (fP3Pos + fP3Neg); }
+  const TVector3 KinePos() const { return  fP3Pos; }
+  const TVector3 KineNeg() const { return  fP3Neg; }
```

Comment:
Could return const TVector3 & in order to avoid unnecessary copying.

Reply:
Thanks for your suggestion. The code is updated.